### PR TITLE
Validate mongo config

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR performs MongoDB parameters validation and test that settings are properly parsed.
This avoid silent errors.

Deprecation warning are now logged too.

### CODE REVIEW

File: udata/models/__init__.py [hunk 1]
```diff
@@ -102,15 +105,38 @@ def resolve_model(self, model):
...
+        config['MONGODB_HOST'] = config['MONGODB_HOST_TEST']
+    else:
+        parsed_url = urlparse(config['MONGODB_HOST'])
+        parsed_url = parsed_url._replace(path='%s-test' % parsed_url.path)
+        config['MONGODB_HOST'] = parsed_url.geturl()
+    validate_config(config)
+
+
 def init_app(app):
     # use `{database_name}-test` database for testing
```
Comment (abulte) [inline udata/models/__init__.py]: Comment needs to be removed I think.

Comment (abulte) [reply udata/models/__init__.py]: Or moved.

**Comment by noirbizarre** [comment]
OK comments moved.

I also initialized logging earlier allowing to log deprecations.

File: udata/app.py [hunk 2]
```diff
@@ -123,7 +123,23 @@ def default(self, obj):
...
+
+
+def init_logging(app):
+    logging.captureWarnings(True)  # Display warnings
+    debug = app.debug or app.config.get('TESTING')
+    log_level = logging.DEBUG if debug else logging.WARNING
+    app.logger.setLevel(log_level)
+    for name in app.config['PLUGINS']:
+        logging.getLogger('udata_{0}'.format(name)).setLevel(log_level)
+    for logger in WARNING_LOGGERS:
```
Comment (abulte) [inline udata/app.py]: Maybe explain why we do that?

Comment (noirbizarre) [reply udata/app.py]: :+1:

Comment (noirbizarre) [reply udata/app.py]: Done (and renamed into `VERBOSE_LOGGERS` to be more explicit

Comment (abulte) [reply udata/app.py]: 👌

**Pull request merged**